src/app/features/collectibles/components/bitcoin/hooks/use-ordinals-render.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+import { useInView } from 'react-intersection-observer';
+
+export function useOrdinalsRender({ fetchNextPage }: { fetchNextPage: () => Promise<void> }) {
```

Comment:
This is more a generic fetch next page fn vs something ordinals specific?

Reply:
here it assumes ordinals specific

Reply:
Where is the ordinals specific code?

Reply:
decided to remove it, seems no need in this hook